scripts/install_eth.cmake

```diff
@@ -0,0 +1,48 @@
+function(download URL DST_FILE STATUS)
```

Comment:
Please could you add the block file header, with the copyright/license, etc?
You can copy-and-tweak it from `install_deps.sh`.   Thanks!